hedyorg/hedy
# 🪲 www.hedycode.com not redirected

When you search Google for Hedy, `www.hedycode.com` is still the first hit instead of `hedy.org`. The reason is we were only redirecting `hedycode.com` without the `www` properly, but not the www-domain.

Also redirect the `www`-domain.

**How to test**

Cannot be tested offline. Deploy first, then visit `www.hedycode.com` and observe that you are redirected.
